etstool.py

```diff
@@ -115,10 +112,8 @@
 }
 
 environment_vars = {
-    "pyside": {"ETS_TOOLKIT": "qt4", "QT_API": "pyside"},
     "pyside2": {"ETS_TOOLKIT": "qt4", "QT_API": "pyside2"},
-    "pyqt": {"ETS_TOOLKIT": "qt4", "QT_API": "pyqt"},
-    "pyqt5": {"ETS_TOOLKIT": "qt4", "QT_API": "pyqt5"},
+    "pyqt": {"ETS_TOOLKIT": "qt4", "QT_API": "pyqt5"},
```

Comment:
I think it is OK to do this, since Pyside2 is the blessed Python Qt wrapper provided by the Qt Company.  If a new wrapper cam along it would likely follow the Pyside conventions more than the PyQt conventions.

Reply:
Post-mortem: It is painful when an old environment name is reused for a new meaning and then that change has to be reverted later. Let's not do this again.